lib/matplotlib/ticker.py

```diff
@@ -1899,7 +1899,7 @@ def view_limits(self, dmin, dmax):
 
 
 def scale_range(vmin, vmax, n=1, threshold=100):
-    dv = abs(vmax - vmin)  # > 0 as nonsingular is called before.
```

Comment:
I think the point is that this is nonzero (>0, not >=0).

Reply:
Ah makes sense - I'll put the comment back